components/blog/Post.tsx

```diff
@@ -0,0 +1,88 @@
+import Image from 'next/image';
+import { getStrapiMedia, formatDate } from '../../utils/blog/api-helpers';
+import { postRenderer } from '../../utils/blog/post-rendered';
+
+export interface Article {
+	id: number;
+	attributes: {
+		title: string;
+		description: string;
+		slug: string;
+		cover: {
+			data: {
+				attributes: {
+					url: string;
+				};
+			};
+		};
+		authorsBio: {
+			data: {
+				attributes: {
+					name: string;
+					avatar: {
+						data: {
+							attributes: {
+								url: string;
+							};
+						};
+					};
+				};
+			};
+		};
+		blocks: any[];
+		publishedAt: string;
+	};
+}
+
+const Post: React.FC<{ article: Article["attributes"] }> = ({ article }) => {
+	const { title, description, publishedAt, cover, authorsBio, blocks } = article;
+	const author = authorsBio.data?.attributes;
+	const imageUrl = getStrapiMedia(cover.data?.attributes.url);
+	const authorImgUrl = getStrapiMedia(authorsBio.data?.attributes.avatar.data.attributes.url);
+
+	if(imageUrl === null || authorImgUrl === null) {
+		console.error(`[blog/Post] imageUrl = ${imageUrl}, authorImgUrl = ${authorImgUrl}`);
+		return <></>;
```

Comment:
Why are we not showing the article at all, just because we weren't able to get the author's image? 
Just show a placeholder dummy image there and continue with displaying the article.

High LOGAF.

Reply:
fixed

Reply:
just removed the return statement. Not added placeholder image yet, adding todo